src/tasks/test/TestBattleToEnd.py

```
@@ -1,32 +1,42 @@
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.name = "测试"
         self.group_name = "工具与调试"
         self.group_icon = FluentIcon.DEVELOPER_TOOLS
+        self.default_config = {self.CFG_STAGE_NAME: stages_list[0]}
+        self.config_type[self.CFG_STAGE_NAME] = {
+            "type": "drop_down",
+            "options": stages_list,
+        }
+
+        # to_end 只依赖这些 DailyBattleFeature 实例状态，避免注册完整刷体力配置。
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)